tests/unit/test_utils.py

```
@@ -346,50 +347,62 @@ def test_incomplete_sequencing_runs_are_skipped(self):
-                    f"{old_run} older than maximum age (96 h) to monitor "
+                    f"{run_dir} older than maximum age (96 h) to monitor "
                     "for upload and will not be uploaded"
                 )
 
                 self.assertIn(expected_log_message, "".join(log.output))
 
+        # test that when the run age is within the max_age it is
+        # selected for upload
+        with self.subTest("testing outputs are present when within max_age"):
+            to_upload, partial_upload = utils.get_runs_to_upload(
+                monitor_dirs=[sequencer_output_dir], max_age=128
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)